libraries/features/dma/simulation/dramsim3.mk

```diff
@@ -64,6 +64,9 @@ $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -I$(BASEJU
 $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -I$(BASEJUMP_STL_DIR)/imports/DRAMSim3/ext/fmt/include
 $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -DFMT_HEADER_ONLY=1
 $(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -DBASEJUMP_STL_DIR="$(BASEJUMP_STL_DIR)"
+$(LIBRARIES_PATH)/features/dma/simulation/libdramsim3.so: CXXFLAGS += -DBLOOD_GRAPH
```

Comment:
remove comments if deprecated?

Reply:
@tommydcjung I guess these comments might be intentional..? We are expecting users to uncomment these lines if they are trying to generate blood graphs.. By default we disable them to reduce the amount of data dumped right?

Reply:
Yes, for now, this is how we turn off blood graph... maybe add a comment explaining that?

Reply:
Comments added!